packages/assistify-help-request/client/views/creationDialog/AssistifyCreateRequest.html

```diff
@@ -48,11 +48,8 @@ <h1 class="create-channel__title">{{_ "Requests"}}</h1>
 							   placeholder={{_ "New_request_title"}} autocomplete="off">
 					</div>
 				</label>
-				{{#if requestTitleInUse}}
-					{{> AssistifyCreateInputError text="Request_already_exists"}}
-				{{/if}}
-				{{#if invalidTitle}}
-					{{> AssistifyCreateInputError text="Request_no_special_char"}}
+				{{#if titleError}}
```

Comment:
having a container for the actual error reduces 'if's

Reply:
@mrsimpson implemented this at first, but having this approach will propagate one message at a time-point. This will prevent highlighting multiple error messages for a input field. In this case, its sufficient to have this approach work as expected.